finalPrep/modZeroHero.js

```diff
@@ -1,40 +1,59 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Samus Mist";
+var specialAbility = "Spider Ball";
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = "I am " + heroName + " and I am here to help you!";
+var catchphrase = "Here comes ${specialAbility}!";
 // Declare two variables - power AND energy - set to integers
-
+var power = 10;
+var energy = 100;
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = 5000;
+var fullEnergy =250;
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["Spiderman", "Batman", "Hulk"];
+var sidekicks = ["Robin", "Antman", "Captain America"];
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
```

Comment:
How would you print the last `archEnemy` to the console if you didn't know the length of the array?